webapp.py

```diff
@@ -0,0 +1,79 @@
+from flask import Flask, render_template, request, redirect, url_for
+from main import RESULT_DIR, FOLDER_TYPES, TARGET_FOLDER
+import lib.arrange
+import lib.utils
+import webbrowser
+import time
+import os
+
+app = Flask(__name__)
+destination = os.path.join(TARGET_FOLDER, RESULT_DIR)
+
+
+def startwork():
+    """Create folders to keep files according to their file types"""
+    destination = os.path.join(TARGET_FOLDER, RESULT_DIR)
+    lib.utils.makeFolders(destination, FOLDER_TYPES.keys())
+
+
+@app.route('/dashboard/<address>')
+def dashboard(address):
+    """Scan the location for items"""
+    dataPacket = {}
+    Elements = list(os.scandir(address))
+    dataPacket['folder_count'] = (len([1 for x in Elements if x.is_dir()]))
+    dataPacket['file_count'] = (len([1 for x in Elements if x.is_file()]))
+    dataPacket['total_items'] = len(Elements)
+    dataPacket['address'] = address
+    return render_template("index.html",data = dataPacket)
+
+
+@app.route('/input', methods=['POST', 'GET'])
+def inputuser():
+    """Take the input of the address from HTML methods"""
+    if request.method == 'POST':
+        val = request.form['newaddress']
+    else:
+        val = request.args.get('newaddress')
+    if os.path.isdir(val):
+        TARGET_FOLDER = str(val)

```

Comment:
This is a bug here. If I enter another location, the TARGET_FOLDER does not change in the global value. The script attempts to clean in the original folder only